.pre-commit-config.yaml

```diff
@@ -101,7 +101,7 @@ repos:
       - id: pyupgrade
         args: ["--keep-percent-format"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.5.1
+    rev: 5.12.0
     hooks:
       - id: isort
         name: isort except __init__.py
```

Comment:
please do this kind of thing in a separate PR in the future, because this we can fasttrack to avoid conflicts and have all other PRs rebase afterwards

Reply:
I rebased.